mail_composer_cc_bcc/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Email CC and BCC",
+    "summary": "This module enables sending mail to CC and BCC partners in mail composer form.",
+    "version": "17.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Social",
+    "website": "https://github.com/OCA/social",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Camptocamp, Odoo Community Association (OCA)",
```

Reply:
this was a request from Simone on migration of module to 16.0:

![image](https://github.com/trisdoan/social/assets/511258/923d84f2-857f-4570-9eed-1210d6a30f34)

somehow forgotten

Reply:
`https://github.com/OCA/social/pull/1224#discussion_r1351694043`